packages/mlcuddidl/mlcuddidl.3.0.2/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "Nicolas Berthier <m@nberth.space>"
+authors: ["Bertrand Jeannet" "Nicolas Berthier"]
+homepage: "https://www.inrialpes.fr/pop-art/people/bjeannet/mlxxxidl-forge/mlcuddidl/index.html"
+bug-reports: "https://gforge.inria.fr/projects/mlxxxidl/"
+#dev-repo: "https://scm.gforge.inria.fr/anonscm/svn/mlxxxidl/mlcuddidl/"
```

Comment:
Why is commented ?

Reply:
This project is hosted on SVN, and `dev-repo' fields does not allow that kind of url (yet: https://github.com/ocaml/opam/issues/2409 ).

Reply:
I see. I didn't know that.